ngx-fudis/projects/ngx-fudis/src/lib/components/form/text-input/readme.mdx

```diff
@@ -17,6 +17,17 @@ Text Input type options are `text` (default), `email`, `number`, `password`, `te
 
 Necessary properties for Text Input are `label` and `control`.
 
+### Autocomplete
+
+`autocompleteFill` will add HTML autocomplete attribute to the input which will then provide automated assistance in filling out form field values.
```

Comment:
Since autocomplete is native HTML attribute, could we just use the same name? At least we are following this naming convention with rest of native attributes.

Reply:
I was trying to add some differentiation to our Select with autocomplete but maybe it's in vain. Will rename it to match the native attribute.